### ISSUE TITLE
Keyboard accessibility - global ticket

### ISSUE THREAD

**Issue by adamkendis** [hackforla/311-data#582]
# Description
Global ticket for all keyboard accessibility tasks. Add/update action items as necessary.

## Action Items
- [x] Filters menu - Implement bypass blocks to skip to next filter.
- [x] Ensure all links and controls can be accessed via the Tab key.
- [x] Ensure no components trap the keyboard. User should be able to get into and out of any UI element.
- [x] Ensure logical tab flow.

Page/component checklist (probably incomplete):
- [x] Header links, including hamburger menu links when menu is collapsed. 
- [x] Filter menu - map/data vis
- [x] Filter menu - comparison tool
- [x] Custom date range picker - tabbing backward to x works, but calendar currently blocks "x" selector
- [x] District selector modal (comparison) - able to tab to plus sign
- [x] Distict selector modal - able to tab to x to close modal
- [x] Map - export button, zoom controls, layer controls, attribution link
- [x] Data visualizations pane - Not able to tab between "map" and "data visualization", need to add skip about nc selection.
- [x] Comparison visualizations pane - Able to tab to "export" but not able to select it
- [x] About page
- [x] Contact page
- [x] StaticFooter - About/Contact pages have links

Focus highlighting is covered by #561 

## Resources
https://jimthatcher.com/skipnav.htm
https://www.csun.edu/universal-design-center/web-accessibility-criteria-bypass-blocks

https://xd.adobe.com/view/74d66153-1ce5-4465-4cbd-66fc40fcc84a-02e7/

**Comment by hannahlivnat** [comment]
@adamkendis I updated the checklist with some of the specific items missing for each page. The majority of content seems keyboard accessible apart from an element or two on each page!

**Comment by adamkendis** [comment]
@hannahlivnat Excellent, thanks! It looks like this ticket will be much easier than we thought.

**Comment by alexchoiweb** [comment]
Hey everyone, I was able to get a few things checked off
-Filters menu skip navigation
-custom date range picker X
-District selector modal tab to +
-District selector modal tab to X

Will continue to work on the rest of the checklist.

**Comment by adamkendis** [comment]
Tweaking appearance/functionality of skip links and adding a few missing keypress handlers. 
ETA tomorrow.
